hdf5-build.patch

```diff
@@ -94,35 +121,23 @@ diff -up hdf5-1.10.5/java/examples/intro/Makefile.am.build hdf5-1.10.5/java/exam
  CLASSPATH_ENV=CLASSPATH=.:$(JAVAROOT):$(top_builddir)/java/src/$(hdfjarfile):$(top_srcdir)/java/lib/slf4j-api-1.7.25.jar:$(top_srcdir)/java/lib/ext/slf4j-simple-1.7.25.jar:$$CLASSPATH
  
  jarfile = jar$(PACKAGE_TARNAME)intro.jar
-diff -up hdf5-1.10.5/java/src/Makefile.am.build hdf5-1.10.5/java/src/Makefile.am
---- hdf5-1.10.5/java/src/Makefile.am.build	2018-09-03 20:54:44.000000000 -0600
-+++ hdf5-1.10.5/java/src/Makefile.am	2019-04-07 15:29:56.308854675 -0600
-@@ -33,8 +33,8 @@ JAVAROOT = .classes
+diff --git a/java/src/Makefile.am b/java/src/Makefile.am
+index b7ad4bd..da3d34f 100644
+--- a/java/src/Makefile.am
++++ b/java/src/Makefile.am
+@@ -33,7 +33,7 @@ JAVAROOT = .classes
  classes:
  	test -d $(@D)/$(JAVAROOT) || $(MKDIR_P) $(@D)/$(JAVAROOT)
  
 -jarfile = jar$(PACKAGE_TARNAME)-$(PACKAGE_VERSION).jar
--hdf5_javadir = $(libdir)
 +jarfile = $(PACKAGE_TARNAME).jar
-+hdf5_javadir = $(prefix)/lib/java
+ hdf5_javadir = $(libdir)
  
```

Comment:
@brianjmurrell 

This code is no longer needed in the .spec file
# Leap 15.1 wants jars in /usr/lib64/java
%if (0%{?suse_version} >= 1500)
ed java/src/Makefile.am << EOF
/^hdf5_javadir =/s/lib/lib64/
wq
EOF
%endif

Notice the change in 1.10 to 1.12 
hdf5_javadir = $(prefix)/lib/java    =>   hdf5_javadir = $(libdir)
Should I change it to hdf5_javadir = $(libdir)/java?

Reply:
This looks like some part(s) of the `hdf5-build.patch` got dropped when it was regenerated.  This updating of hdf5_javadir is the distribution adjusting the upstream build to fit it's filesystem layout standard.  So this change should not have been dropped from the patch.

Here's the distro's patch for reference: https://src.fedoraproject.org/rpms/hdf5/blob/master/f/hdf5-build.patch

I think I am going to have to review this new regenerated version of this patch more closely as I missed this droppage.

Reply:
@brian  I scrubbed through the patch files again and I did miss this in the src/java/Makefile.am
I verified that it was updated to: (centos)
jarfile = $(PACKAGE_TARNAME).jar
hdf5_javadir = $(prefix)/lib/java

Reply:
So is there a new commit to this PR that I'm not seeing that resolves this?